src/Components/ReviewCard.jsx

```diff
@@ -0,0 +1,9 @@
+export const ReviewCard = ({ title, designer, owner, review_img_url, review_body, category, created_at, votes}) =>{
+    return(
+        <li className="ReviewCard" >
+            <p className="reviewTitle">{title}</p>
+            <img className="reviewImage" src={`${review_img_url}`} alt={`${title}`} />
```

Comment:
Not hurting anyone, but you don't need template literals here eg `src={review_img_url}`